port/win/xpress_win.cc

```diff
@@ -202,11 +202,69 @@ char* Decompress(const char* input_data, size_t input_length,
   return outputBuffer.release();
 }
 
+int64_t GetDecompressedSize(const char* input_data, size_t input_length) {
```

Comment:
Can be used by DecompressToBuffer() too?

Reply:
DecompressToBuffer doesn't need it